src/Core/Components/NavMenu/FluentNavLink.razor

```diff
@@ -47,7 +47,7 @@ else if (!Owner.Expanded && Owner.CollapsedChildNavigation && SubMenu != null)
         @if (Icon is not null)
         {
             <span slot="start">
-                <FluentIcon Value="@Icon" Color="@IconColor" Class="fluent-nav-icon" />
+                <FluentIcon Value="@Icon" Color="@IconColor" CustomColor="@CustomColor" Class="fluent-nav-icon" />
```

Comment:
I think you need to make the same changes in the `FluentNavGroup.razor` file.

Reply:
Will do for FluentNavGroup also when I update the other.  Hopefully before the weekend.  Will see.